### ISSUE TITLE
Update HTML to support accessibility

### ISSUE THREAD

**Issue by trynthink** [trynthink/scout#37]
Bootstrap includes in their documentation several notes regarding how to configure various page elements to support accessibility. These changes should be incorporated throughout.

**Comment by trynthink** [comment]
In working on this issue, I've encountered some useful resources beyond the Bootstrap documentation. 

* VoiceOver, built into Mac OS X, is a useful tool for testing how screen readers will perform on a given page.VoiceOver is activated with ⌘-F5. A handy overview of the primary key commands to use VoiceOver can be found on [this website](http://webaim.org/articles/voiceover/#navigation). 
* The Google Developers team created a [long but useful video](https://youtu.be/x18vEEfpK3g) that covers issues related to accessibility.

Keep in mind that accessibility covers not just those who use a screen reader with aural or Braille output, but also those who cannot use a mouse. Those who are deaf or hard-of-hearing are also generally included, but since our site has no audio or video content, no particular modifications are required for those users.
